module_raid.lua

```diff
@@ -7,6 +7,7 @@ local discordia = Discordia
 local bot = Bot
 local enums = discordia.enums
 local band, bor, bnot = bit.band, bit.bor, bit.bnot
+local moduleSpam = require('module_spam')
```

Comment:
This will enable the `module_spam` module even if it is disabled in the guild if I am not mistaken. I don't think it's a good idea to do modules dependances.